content/intro/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+linkTitle: अवलोकन
+title: Brigade की अवलोकन
+description: Brigade का उच्च स्तरीय दृश्य
+section: परिचय
+weight: 1
+aliases: 
+  - /overview.md
+  - /intro/overview.md
+---
+
+Brigade एक इवेंट संचालित स्क्रिप्टिंग टूल है जो Kubernetes पर चलता है। इसका मतलब यह है:
+
+- Brigade क्लाउड में स्क्रिप्ट योग्य स्वचालित कार्यों को चलाने के लिए है।
+- Brigade के लिए आपको कोई होस्ट सर्वर प्रबंधित करना ज़रुरी नहीं है।
+-Brigade विशेष रूप से सीआई और सीडी वर्कलोड के लिए उपयुक्त है जैसे:
+- Automated testing
+  - GitHub hook integration
+  - Building artifacts and releases
+  - Managing deployments
+- Brigade अपने API server के साथ रहता है |
+  - बातचीत सीधे ब्रिगेड के माध्यम से होती है, `brig` CLI या SDK के माध्यम से
```

Comment:
बातचीत != interaction. Also, let's use सीएलआई and एसडीके instead.

Reply:
What word can be used instead of बातचीत for interactions? Can we just write इन्टरैक्शन in that place?

Reply:
Yes, let's transliterate.